spec/apibuilder-spec.json

```diff
@@ -218,7 +218,8 @@
 	{ "name": "type", "type": "string" },
 	{ "name": "headers", "type": "[header]", "required": false },
 	{ "name": "description", "type": "string", "required": false },
-	{ "name": "deprecation", "type": "deprecation", "required": false }
+	{ "name": "deprecation", "type": "deprecation", "required": false },
+	{ "name": "attributes", "type": "[attribute]", "default": [] }
```

Comment:
Not entirely sure, but this may be a breaking change, because existing apibuilder files don't have this value. Field with default value are implicitly required on the wire, so the safer option I thinknwould be to make it  "required": false.

Reply:
Good point.  That seems like how it should work, but then shouldn't all the other tests in ServiceResponseSpec fail?

Reply:
good catch @gheine - yes need to make optional.

not sure what other tests @ericmittelhammer - also maybe attributes existed from the beginning (vs. being added post launch)

Reply:
Ack.  will make optional.

and I think the reason that the unit tests pass without providing attributes on a response is because I'm providing a default value in the form:  https://github.com/apicollective/apibuilder/pull/697/files#diff-f0b103452b2b11b4f5744d8b56d5e9beR272